code/turf/walls.dm

```diff
@@ -342,18 +331,6 @@
 		else return
 
 	else
-		if(src.material)
-			src.material.triggerOnHit(src, W, user, 1)
```

Comment:
why remove the matProc trigger?

Reply:
Oops didnt really think that one through Ill readd it

Reply:
Done! Ditto for other